src/components/UI/Input/Input.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const Input: React.FC<React.InputHTMLAttributes<HTMLInputElement>> = (
+  props,
+) => {
+  return (
+    <input {...props} />
+  );
+};
```

Comment:
and why do we need separate component for the input if we just pass all props there